### PR TITLE
feat: Clear minio s3 bucket

### PR DESCRIPTION
**Pull request by Quentinchampenois**
#### :tophat: Description
*Please describe your pull request.*

- [x] Add MinIO S3 Bucket in docker compose
- [x] Clear Orphans from database
- [x] Clear Orphans from bucket
- [ ] Add specs for ClearOrphanJob
- [ ] Add specs for PurgeBlobsJob

### CODE REVIEW

**Comment by Quentinchampenois** [comment]
GitGuardian alert is false positive, it was fake generated secrets. I changed them to prevent another GitGuardian trigger

**Review by Stef-Rousset — APPROVED** [review]
LGTM

File: lib/tasks/active_storage.rake [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      Rails.logger.info "Looking for blobs without attachments in database..."
+      blobs = ActiveStorage::Blob.where.not(id: ActiveStorage::Attachment.select(:blob_id))
+
+      if blobs.count.zero?
+        Rails.logger.info "Database is clean !"
+        Rails.logger.info "Terminating task..."
+      else
+        Rails.logger.info "Found #{blobs.count} orphan blobs !"
+
+        ActiveStorage::Blob.where.not(id: ActiveStorage::Attachment.select(:blob_id)).find_each(&:purge)
```
Comment (Quentinchampenois) [inline lib/tasks/active_storage.rake:16]: Duplicated SQL query, should be memoized

**Review by Quentinchampenois — COMMENTED** [review]
Must optimize rake task and even move to dedicated Job

**Pull request merged**